app/services/activities/import_from_csv.rb

```diff
@@ -17,10 +17,10 @@ def column_to_csv_column_mapping
     attr_reader :errors, :created, :updated
 
     def self.column_headings
-      ["Parent RODA ID"] + Converter::FIELDS.values - ["BEIS ID"]
+      ["Parent RODA ID"] + Converter::FIELDS.values - ["BEIS ID"] + ["Comments"]
```

Comment:
Consider adding Comments to Converter::FIELDS

Reply:
We can't trivially mass-assign Comments, because it's structurally different to all the other fields.
So it probably doesn't belong in that big list?